microsite/data/plugins/docker-tags.yaml

```
@@ -0,0 +1,10 @@
+---
+title: Docker tags
+author: Workm8
+authorUrl: 'https://github.com/work-m8/.github'
```

Do you actually want to redirect to the repository? Or the organisation? :)

```suggestion
authorUrl: 'https://github.com/work-m8'
```